test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/no/no/no/no/no/yes.txt

```diff
@@ -0,0 +1,14 @@
+
+
+You must [check with the Home Office](https://eforms.homeoffice.gov.uk/outreach/lcs-application.ofml) to find out if the tenant is allowed to rent property. You’ll need the person’s Home Office reference number.
+
+You’ll get an answer within 2 working days.
+
+If the person’s allowed to rent, the Home Office will tell you if you need to check their documents again later.
```

Comment:
@ikennaokpala I think that `No 2` is what will become `No 3` in the logic document. Polly has already agreed to update the logic document with the new template text.